woocommerce/woocommerce-android
# Widget/analytics

Closes: #7384


### Description
This PR adds a track event when a user taps on today's stats widget and adds a property to the `*_application_opened` event to track the WooCommerce widgets installed by the user.

### Testing instructions
TC1

1. Open the WooCommerce app 
2.  Check that the app sent:

```
🔵 Tracked: application_opened, Properties: {"widgets":[],"is_debug":true}
```

TC2

1. Add a Today's stats widget to the home screen
2. Open the app
3. Check that the app sent:

```
🔵 Tracked: application_opened, Properties: {"widgets":["today-stats"],"is_debug":true}
```

TC3

1. Tap on the Today's stats widget added to the home screen
2. Check that the app sent:

```
🔵 Tracked: widget_tapped, Properties: {"name":"today-stats","blog_id":178322510,"is_wpcom_store":false,"is_debug":true}
```

TC4

1. Remove the Today's stats widget added to the home screen
2. Open the WooCommerce app 
3.  Check that the app sent (no widgets on the widgets property):

```
🔵 Tracked: application_opened, Properties: {"widgets":[],"is_debug":true}
```
